src/lib/course.ts

```diff
@@ -181,7 +181,11 @@ export async function exportStudentCSV(courseId: string, studentId: string): Pro
     const authHeaders = {
       'Authorization': `Bearer ${token}`
     }
-    return await getJson(`https://octopus.${domain}/api/v1/courses/${courseId}/students/${studentId}/export/csv`, undefined, authHeaders)
+    return await getJson(
+        `https://octopus.${domain}/api/v1/courses/${courseId}/students/${studentId}/export/csv`,
```

Comment:
Api paths starts from the same string, I think it is better to move constructing the first part to some helper function - it will be easier to change it during url changes